doc/lsp.txt

```diff
@@ -507,6 +507,10 @@ outlineOnRight		|Boolean| option.  Open the outline window on the
 						*lsp-opt-outlineWinSize*
 outlineWinSize		|Number| option.  The size of the symbol Outline
 			window.  By default this is set to 20.
+						*lsp-opt-propOverride*
+propOverride		|Boolean| option.  when TRUE textprops highlight
+			overrides any other, including 'cursorline' and Visual
+			By default this is set to false.
```

Comment:
if make it judge/work with `cursorlineopt` then no this option was required, it is 'ture' when no cursorline.
and wish it is default true, otherwise user maybe thought there is no diag (but actually it just been coverred).
but simply it is ok, i think we can live with this option.

Reply:
I made the default for `propOverride` equal:

```viml
&cursorline && &cursorlineopt =~ '\<line\>\|\<screenline\>\|\<both\>'
```

Reply:
if you insisted this option required, then i think better be leave that cursorlineopt alone, just let user manager it via this option explicitly: which is directly and simply: 'true' or 'false'.

Reply:
It will be simpler from a user perspective, if this option is set to either true or false (instead of a Boolean expression).  Can you move the check for the `cursorline` and `cursorlineopt` into the plugin code?

Reply:
Like this: https://github.com/yegappan/lsp/pull/326 ?